numba_dpex/core/runtime/CMakeLists.txt

```diff
@@ -0,0 +1,124 @@
+#[=======================================================================[.rst:
+_dpexrt_python
+---------------
+
+A cmake file to compile the ``_dpexrt_python`` Python C extension for
+``numba_dpex``. You can build this component locally in-place by invoking these
+commands:
+
+.. code-block:: cmake
+    ~$ cmake .
+    ~$ cmake --build . --verbose
+
+Once compiled, the _dpexrt_python library will be in ``numba_dpex/core/runtime``
+folder.
+#]=======================================================================]
+
+cmake_minimum_required(VERSION 3.21...3.27 FATAL_ERROR)
+
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.24")
+    cmake_policy(SET CMP0135 NEW)
+endif()
+
+project(_dpexrt_python
+    DESCRIPTION "A Python C extension for numba-dpex runtime."
+)
+
+# Get numba include path
+if(NOT DEFINED Numba_INCLUDE_DIRS)
+    execute_process(
+        COMMAND python -c "import numba; print(numba.extending.include_path());"
+        OUTPUT_VARIABLE Numba_INCLUDE_DIRS
+        RESULT_VARIABLE RET
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+
+    if(RET EQUAL "1")
+        message(FATAL_ERROR "Module \'numba\' not found.")
+    endif()
+endif()
+
+# Get dpctl library path
+if(NOT DEFINED DPCTL_LIBRARY_PATH)
```

Comment:
How about using the `FindDpctl` cmake macro? https://github.com/IntelPython/dpctl/blob/master/cmake/FindDpctl.cmake

Reply:
We are actually using the macro on line 84. Where do we need `DPCTL_LIBRARY_PATH`?

Reply:
That macro doesn't set `DPCTL_LIBRARY_PATH` so we had to set it manually.